src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/FallbackRazorProjectHost.cs

```diff
@@ -125,7 +125,7 @@ internal async Task OnProjectChanged(IProjectVersionedValue<IProjectSubscription
                     }
 
                     var configuration = FallbackRazorConfiguration.SelectConfiguration(version);
-                    var hostProject = new HostProject(CommonServices.UnconfiguredProject.FullPath, configuration);
+                    var hostProject = new HostProject(CommonServices.UnconfiguredProject.FullPath, configuration, rootNamespace: null);
```

Comment:
I'm not setting `RootNamespace` here. I assumed we don't care about `RootNamespace` in unconfigured project case.

Reply:
What's the user-visible behavior of that?

Reply:
The design time code gen will be generated with namespace `__GeneratedComponent` instead of the RootNamespace. So intellisense for all components will be available regardless of the folder structure. What I am not sure of is in what cases `FallbackRazorConfiguration` be used for Component apps?

Reply:
Only time this will ever impact components is if something is severely misconfigured in a users app and we can't determine what type of project it is. IMO not something we should worry about because this is a severe error case.

Reply:
Yeah, I think this is OK.